src/components/trainingInput/TrainingInput.module.css

```
@@ -0,0 +1,3 @@
+label {
```

In module CSS, you need to work with dot notations, so you also need classNames in your `TrainingInput.tsx`.

```suggestion
.label {
```